components/automate-ui/src/app/modules/desktop/insight/insight.component.ts

```diff
@@ -39,4 +41,12 @@ export class InsightComponent {
   public sortOn(fieldName: string): void {
     this.sortChange.emit(fieldName);
   }
+
+  public toggleAttributeMenu(): void {
+    this.attributesMenuOpen = !this.attributesMenuOpen;
+  }
+
+  public updateFilters(event) {
+    console.log(event);
```

Comment:
@SEAjamieD are we temporarily keeping the console.log?

Reply:
Yeah that is my intention.  This is being built so that someone can plug it into the actual dashboard with real filter calls.  I'm not sure whether that will end up being me, you, or Scott, but I thought that it might be helpful in guiding whoever uses this next that the updateFilters function is already available.